From 50f1442edb2d2404b0f85099f2873099d2ba1658 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Thu, 23 Aug 2018 15:20:37 +0900
Subject: [PATCH 518/909] arm64: dts: salvator-common: Add no use ddc option
 for VGA connector

commit 1b87624e87420c229af34738e888d9095f8aef74 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

By setting this option, change to not use DDC (Display Data Channel)
on the VGA connector.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/salvator-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/salvator-common.dtsi b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
index ce57919c0afb..a538fe295d9c 100644
--- a/arch/arm64/boot/dts/renesas/salvator-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
@@ -204,6 +204,7 @@
 
 	vga {
 		compatible = "vga-connector";
+		no-use-ddc;
 
 		port {
 			vga_in: endpoint {
-- 
2.17.1

